dll/win32/browseui/shellbrowser.cpp

```diff
@@ -3676,6 +3675,18 @@ LRESULT CShellBrowser::OnToggleStatusBarVisible(WORD wNotifyCode, WORD wID, HWND
     return 0;
 }
 
+LRESULT CShellBrowser::CreateStatusBar()
+{
+    if (!fStatusBar)
+    {
+        fStatusBar = CreateWindowExW(0, STATUSCLASSNAMEW, NULL, WS_CHILD | WS_VISIBLE |
+                                     WS_CLIPSIBLINGS | SBARS_SIZEGRIP | SBARS_TOOLTIPS,
+                                     0, 0, 500, 20, m_hWnd, (HMENU)FCIDM_TB_UPFOLDER,
+                                     _AtlBaseModule.GetModuleInstance(), 0);
+    }
+    return 0;
```

Comment:
This `return` value looks useless.

Reply:
This return value is required to use the standard ```LRESULT``` return value. Using a plain void is bad practice in Win32.

Reply:
Alternatively the function could return a BOOL that is TRUE in case the status bar creation succeeded, and FALSE if not.